domain/src/DomainLoaderInterface.php

```diff
@@ -30,7 +29,7 @@ public function load($id, $reset = FALSE);
   /**
    * Gets the default domain object.
    *
-   * @return Drupal\domain\DomainInterface | NULL
+   * @return \Drupal\domain\DomainInterface | NULL
```

Comment:
This comes from us. See [loadDefaultDomain()](https://github.com/agentrickard/domain/blob/8.x-1.x/domain/src/DomainLoader.php#L71)

I did a quick search on the use of `loadDefaultDomain()` and it is mostly checking that it returns something. Bar `DomainSubscriber::onKernelRequestDomain()`